control/settings/controllers/settings.home.controller.js

```diff
@@ -11,6 +11,12 @@
 
         function updateMasterItem(data) {
           SettingsHome.masterInfo = angular.copy(data);
+          if (SettingsHome.masterInfo.data.settings.toggleEmployeeCode == 'on') {
```

Comment:
why are we using 'on' and 'off' instead of boolean values? 
Not saying one is right or wrong, just wondering why?

Reply:
Just following the logic behind initial codebase: 

```
"defaultView": "list",
                                  "distanceIn": "mi",
                                  "mapView": "show",
                                  "filterPage": "show"....

```
this was what I found, so I continued with that, I hope it's ok